pennylane/data/dataset.py

```diff
@@ -158,11 +166,11 @@ def attrs(self):
         return {k: v for k, v in vars(self).items() if k[0] != "_"}
 
     @staticmethod
-    def _read_file(filepath):
-        """Read data from a saved file.
+    def get_file_contents(filepath):
+        """Reads data from a saved file.
```

Comment:
We should specify that the files should be previously created or saved `.dat` one.